varsha.css

```diff
@@ -0,0 +1,119 @@
+.container {
+    height: 5200px;
```

Comment:
Does the height and width are necessary for `conatiner` plese remove and check once

Reply:
yeah we don't need the height and breadth for the container